pkg/auth/authproviders/openshift/backend_impl.go

```diff
@@ -31,11 +31,11 @@ const (
 	// serviceOperatorCAPath points to the secret of the service account, which within an OpenShift environment
 	// also has the service-ca.crt, which includes the CA to verify certificates issued by the service-ca operator.
 	// This could be i.e. the default ingress controller certificate.
-	serviceOperatorCAPath = "/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+	serviceOperatorCAPath = "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
 	// internalServicesCAPath points to the secret of the service account, which includes the internal CAs to
 	// verify internal cluster services.
 	// This could be i.e. the openshiftAPIUrl or other internal services.
-	internalServicesCAPath = "/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+	internalServicesCAPath = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
```

Comment:
Did this ever worked if a false path was configured here? 🤔

Reply:
Yes, `/var/run` is a symlink to `/run` in our central images. As Kubernetes mounts the configMap to `/var/run`, and that's how it is [documented](https://kubernetes.io/docs/tasks/run-application/access-api-from-pod/#accessing-the-api-from-within-a-pod), we decided to rename the paths.